resources/js/components/dashboard/DashboardModules.vue

```diff
@@ -10,7 +10,6 @@
 </template>
 
 <script>
-import { changeRouteTitle } from './../../mixins/changeRouteTitle.js'
 import DashboardNotification from './DashboardNotification'
 
 export default {
```

Comment:
make the module names consistent by capitalizing the first letter of each module name